.github/workflows/pipeline.yml

```
@@ -0,0 +1,193 @@
+      packages: read
+      security-events: write
+    strategy:
+      fail-fast: false
+      matrix:
+        version: ${{ fromJson(needs.detect-versions.outputs.versions) }}
+    steps:
+      - uses: actions/checkout@9c091bb21b7c1c1d1991bb908d89e4e9dddfe3e0 # v7
+
+      - name: Set environment variables
+        id: vars
+        run: |
```

actionlint (1.7.12)

[error] shellcheck reported issue in this script: SC2086:info:5:32: Double quote to prevent globbing and word splitting

(shellcheck)